Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -360,12 +361,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $showProgress = $resolver->getProgress();
 
         if ($showProgress) {
-            $fileProcessedEventListener = function (FixerFileProcessedEvent $event) use ($output) {
-                $output->write($event->getStatusAsString());
-            };
-
             $this->fixer->setEventDispatcher($this->eventDispatcher);
-            $this->eventDispatcher->addListener(FixerFileProcessedEvent::NAME, $fileProcessedEventListener);
+            $stdErr = fopen('php://stderr', 'w');
```

Comment:
I can change that :)
I went for this to make it more easy to change the stream used in the future, but that might not be needed indeed.